common/membership/grpc_resolver.go

```diff
@@ -137,7 +113,7 @@ func (m *grpcResolver) resolve() {
 		})
 	}
 	if err := m.cc.UpdateState(resolver.State{Addresses: addresses}); err != nil {
-		fmt.Printf("error updating state in gRPC resolver: %v", err)
+		fmt.Printf("error updating state in gRPC resolver: %v\n", err)
```

Comment:
since we're passing in a Monitor now, we might as well pass in a Logger and log properly